scripts/check

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh -e
+
+export PREFIX=""
+if [ -d 'venv' ] ; then
+    export PREFIX="venv/bin/"
+fi
+export SOURCE_FILES="uvicorn tests"
+
+set -x
+
+${PREFIX}black --check --diff --target-version=py36 $SOURCE_FILES
+${PREFIX}flake8 $SOURCE_FILES
+#${PREFIX}mypy $SOURCE_FILES
```

Comment:
I suppose we can remove this commented line and add it if/when mypy gets added to the repo.

Reply:
done